### PR TITLE
CDAP-8196 Cleanup old log saver classes

### PR DESCRIPTION
**Pull request by chtyim**
- The first commit is purely classes removal.
- Second commit is actually refactoring that just tag along instead of opening another PR...

### CODE REVIEW

**Comment by chtyim** [comment]
Build: http://builds.cask.co/browse/CDAP-RUT558

**Comment by chtyim** [comment]
Build passed: http://builds.cask.co/browse/CDAP-RUT558-2

File: cdap-watchdog/src/main/java/co/cask/cdap/logging/guice/LogSaverServiceModule.java [hunk 1]
```diff
@@ -1,54 +0,0 @@
...
-import com.google.inject.name.Names;
-
-/**
- * Module for LogSaver
- */
-public class LogSaverServiceModule extends PrivateModule {
-
-  @Override
-  protected void configure() {
-    Multibinder<KafkaLogProcessorFactory> logProcessorBinder = Multibinder.newSetBinder
```
Comment (CuriousVini) [inline cdap-watchdog/src/main/java/co/cask/cdap/logging/guice/LogSaverServiceModule.java:40]: we should also remove `KafkaLogProcessor` interface right?

Comment (chtyim) [reply cdap-watchdog/src/main/java/co/cask/cdap/logging/guice/LogSaverServiceModule.java:40]: CAn't yet. In fact Standalone mode is using it. I am refactoring that part of the code, but that would be a separate PR.

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java [hunk 2]
```diff
@@ -533,6 +537,10 @@
     public static final String MESSAGING_TOPIC_NUM = "metrics.messaging.topic.num";
     public static final String MESSAGING_FETCHER_LIMIT = "metrics.messaging.fetcher.limit";
 
+    public static final Map<String, String> METRICS_PROCESSOR_CONTEXT =
```
Comment (CuriousVini) [inline cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:540]: why do we put context in Constants?

**Review by CuriousVini — APPROVED** [review]
minor question, changes look good.

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java (hunk 2 above)
Comment (chtyim) [reply cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java:540]: It is just a constants used in multiple places.

**Comment by chtyim** [comment]
Merging since unit-test passed

**Pull request merged**